src/components/organisms/SettingsChange.tsx

```diff
@@ -0,0 +1,303 @@
+  all: "unset",
+  backgroundColor: "white",
+  width: 25,
+  height: 25,
+  borderRadius: 4,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  boxShadow: `0 2px 10px ${blackA.blackA7}`,
+  "&:hover": { backgroundColor: indigo.indigo3 },
+});
+
+const StyledIndicator = styled(CheckboxPrimitive.Indicator, {
+  color: indigo.indigo11,
+});
+
+// Exports
+export const Tabs = StyledTabs;
+export const TabsList = StyledList;
+export const TabsTrigger = StyledTrigger;
+export const TabsContent = StyledContent;
+export const Checkbox = StyledCheckbox;
+export const CheckboxIndicator = StyledIndicator;
+
+const Box = styled("div", {
+  display: "flex",
+  justifyContent: "center",
+});
+const Flex = styled("div", { display: "flex" });
+
+const Text = styled("div", {
+  marginBottom: 20,
+  color: mauve.mauve11,
+  fontSize: 15,
+  lineHeight: 1.5,
+});
+
+const Annotation = styled("p", {
+  marginTop: 5,
+  color: mauve.mauve11,
+  fontSize: 12,
+  lineHeight: 1.5,
+});
+
+const Button = styled("button", {
+  all: "unset",
+  display: "inline-flex",
+  alignItems: "center",
+  justifyContent: "center",
+  borderRadius: 4,
+  padding: "0 15px",
+  fontSize: 15,
+  lineHeight: 1,
+  fontWeight: 500,
+  height: 35,
+
+  variants: {
+    variant: {
+      indigo: {
+        backgroundColor: indigo.indigo4,
+        color: indigo.indigo11,
+        "&:hover": { backgroundColor: indigo.indigo5 },
+        "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo7}` },
+      },
+    },
+  },
+
+  defaultVariants: {
+    variant: "indigo",
+  },
+});
+const Fieldset = styled("fieldset", {
+  all: "unset",
+  marginBottom: 15,
+  width: "100%",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "flex-start",
+});
+
+const Label = styled("label", {
+  fontSize: 13,
+  lineHeight: 1,
+  marginBottom: 10,
+  color: indigo.indigo12,
+  display: "block",
+});
+
+const Input = styled("input", {
+  all: "unset",
+  flex: "1 0 auto",
+  borderRadius: 4,
+  padding: "0 10px",
+  fontSize: 15,
+  lineHeight: 1,
+  color: indigo.indigo11,
+  boxShadow: `0 0 0 1px ${indigo.indigo7}`,
+  height: 35,
+  "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo8}` },
+  "&::placeholder": { color: mauve.mauve7 },
+});
+
+const FieldCheck = styled("fieldset", {
+  all: "unset",
+  width: "100%",
+  display: "flex",
+  alignItems: "center",
+});
+
+const CheckLink = styled("a", {
+  color: indigo.indigo11,
+  textDecoration: "underline",
+  "&:visited": { color: indigo.indigo11 },
+});
+
+type Props = {
+  setSettingsView: React.Dispatch<React.SetStateAction<string>>;
+  settingsView: React.Dispatch<React.SetStateAction<string>>;
+  // selectedDeviceName: React.Dispatch<React.SetStateAction<string>>;
+  // selectedDeviceIndex: React.Dispatch<React.SetStateAction<number>>;
+  deviceData: React.Dispatch<React.SetStateAction<[]>>;
+  setDeviceData: React.Dispatch<React.SetStateAction<[]>>;
+  userData: React.Dispatch<React.SetStateAction<{}>>;
```

Comment:
Please import UserData that defined at settings page and set UserData here.
`import { UserData } from "@/pages/settings";`
`userData: UserData;`